examples/rust/raw_mesh/src/main.rs

```diff
@@ -50,8 +50,14 @@ impl From<GltfPrimitive> for Mesh3D {
         }
         if albedo_factor.is_some() {
             mesh = mesh.with_mesh_material(rerun::datatypes::Material {
-                albedo_factor: albedo_factor
-                    .map(|[r, g, b, a]| ecolor::Rgba::from_rgba_unmultiplied(r, g, b, a).into()),
+                albedo_factor: albedo_factor.map(|[r, g, b, a]| {
+                    rerun::Rgba32::from_unmultiplied_rgba(
+                        (r / 255.0) as u8,
+                        (g / 255.0) as u8,
+                        (b / 255.0) as u8,
+                        (a / 255.0) as u8,
+                    )
```

Comment:
this looks like a gamma error to me

Reply:
Actually, it looks like more than a gamma error - it will convert white to `#010101`, which is very non-white

Reply:
yeah it's just very wrong all the way